server/irmaserver/sessions.go

```diff
@@ -132,7 +128,10 @@ func (s *memorySessionStore) deleteExpired() {
 	expired := make([]irma.RequestorToken, 0, len(toCheck))
 	for token, session := range toCheck {
 		session.Lock()
-		timeout := maxSessionLifetime
+		timeout := time.Duration(s.conf.MaxSessionLifetime) * time.Minute
+		if s.conf.MaxSessionLifetime == 0 {
+			timeout = 5 * time.Minute
+		}
```

Comment:
Instead of making this decision each time during runtime, would it not be better to do this during verification of the validity of the configuration, during server startup? That is, within `server.Configuration.Check()`, probably in a new helper. Some of the other helpers there make similar modifications to the configuration (e.g. `verifyURL()`).

Reply:
Implemented in server.configuration.check. Did not need helper as there is no need to generate an error.